kubernetes-setup/main.yaml

```diff
@@ -102,11 +91,33 @@
    
     - name: Enable CRI-O
       shell: dnf module enable -y cri-o:1.18
+      when: ansible_distribution == "Fedora"
 
-    - name: Install CRI-O
-      dnf: 
+    - name: Install CRI-O on Fedora
+      dnf:
         name: cri-o
         state: present
+      when: ansible_distribution == "Fedora"
+
+    - name: Build & Install CRI-O on Ubuntu
```

Comment:
How about using the `kubic` repos for Ubuntu as mentioned here - https://cri-o.io/ ?

Reply:
We could use them on x86-64, but those repos are not providing ppc64le binaries for Ubuntu (see http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.21/xUbuntu_20.04/ for example).

Reply:
Ah ok..